danielsaidi/RichTextKit
# [BUGFIX] Minor UI tweaks

# What this PR do:
- Fixes colors for buttons and pickers for dark and light mode 
- Fixes number of Color pickers
# How to test it
- Run app, check if it fits the screenshots

Related: https://github.com/danielsaidi/RichTextKit/issues/125

# How it looks now: 
| macOS  | iOS |
|--------|--------|
| ![PR 1](https://github.com/danielsaidi/RichTextKit/assets/17381941/0fe98fa4-5071-4ff2-8332-0500a5f440c5) | <img width="547" alt="PR 4" src="https://github.com/danielsaidi/RichTextKit/assets/17381941/9a449549-a636-4e1c-95bf-4b173d122040">|
| ![PR 2](https://github.com/danielsaidi/RichTextKit/assets/17381941/b64cfb96-57ca-49c3-afb2-9c0a3544fb65)| <img width="547" alt="PR 3" src="https://github.com/danielsaidi/RichTextKit/assets/17381941/903b4060-ffbb-41ad-933f-ed611c001b93"> |
